src/bootstrap/configure.py

```diff
@@ -178,6 +178,11 @@ def err(msg):
     print("configure: error: " + msg)
     sys.exit(1)
 
+def is_value_list(key):
+    for option in options:
+        if option.name == key and option.desc.startswith('List of'):
+            return True
```

Comment:
Any better way to test a value should be a `list`? 🤔

Reply:
I think this route is not the worse, but make sure all options that are lists are matched.
I saw the `dist-compression-formats` option (line 152), but there could be more.

Reply:
Fixed more options in List.